bin/sendAlertStream.py

```diff
@@ -21,6 +22,29 @@ def load_stamp(file_path):
     return cutout_dict
 
 
+@asyncio.coroutine
+def delay(wait_sec, function):
+    """Sleep for a given time before calling a function.
+    """
+    print('delay starting: {}'.format(time.time()))
```

Comment:
Yeah, think it's fine.  I might cut that later or if not use it when I cross some streams in which case it will make it easier to do math without worrying about formatting.